### DISCUSSION TITLE
Missing languages on my repo, missing Java and Rust

### DISCUSSION THREAD

**Discussion by bellimeow** [github-linguist/linguist#6273] [Classification] [answered]
Hello! I uploaded my repository and only python is showing as a language. Java and rust are missing. Tried to read through the documentation but I can't identify the problem. Any idea how to solve this? Thank you in advance :)

https://github.com/bellimeow/TDP024-enterprise-systems

**Comment by lildude** [comment, marked as answer]
This is happening because over 98% of your repo is Python and this is drowning out the rest of the languages so they're all being bunched into "Other":

```console
➜ github-linguist TDP024-enterprise-systems/
98.71%  11730810   Python
0.98%   116400     Java
0.20%   23695      PowerShell
0.06%   7083       C
0.04%   5261       Rust
0.01%   1348       Batchfile
➜ 
```

Most of these files are in the `API/.venv` directory.

There are several solutions:

1. Remove the `API/.venv` directory from your repo
2. Add a `linguist-vendored` override to a `.gitattributes` file in your repo

**Reply by bellimeow** [reply]
Thank you! Solved the problem by removing the APIs/.venv folder.
